StRoot/StdEdxY2Maker/StdEdxY2Maker.cxx

```diff
@@ -169,8 +169,8 @@ Int_t StdEdxY2Maker::InitRun(Int_t RunNumber){
 
   if (! DoOnce) {
     DoOnce = 1;
-    if ((GetDate() > 20171201 && m_TpcdEdxCorrection->IsFixedTarget()) ||
-	(GetDate() > 20181201)) fUsedNdx = kTRUE; // use dN/dx for fixed target for Run XVIII and year >= XIX
+    if (IAttr("CalcdNdx") && ((GetDate() > 20171201 && m_TpcdEdxCorrection->IsFixedTarget()) ||
```

Comment:
If it was on by default, this would read something like
```suggestion
    if ((!IAttr("DontCalcdNdx")) && ((GetDate() > 20171201 && m_TpcdEdxCorrection->IsFixedTarget()) ||
```

Reply:
A double negation... How about 

```
GetOption("CalcdNdx") ? mk->SetAttr("CalcdNdx", kTRUE) : mk->SetAttr("CalcdNdx", kFALSE);
```

in  StRoot/StBFChain/StBFChain.cxx instead? Would this work?

Reply:
Well, double negation is the whole point of "unless disabled do X" statements.
BFC should support turning option off with `-CalcdNdx`, but then you need to somehow enable it by default.

Reply:
The new commit message reads "By default the calculation is off", so this is formally correct – good to mark as resolved.

Not sure if there is a policy that would prevent a sudden change of the default settings. IMHO it would be nice if there was one.